.travis.yml

```diff
@@ -60,6 +60,17 @@ matrix:
     - compiler: clang
     - os: osx
   include:
+    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_PIRANHA="yes"
+      compiler: gcc
+      os: linux
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          - boost-latest
+          packages:
+          - libboost-serialization1.55-dev
+          - g++-4.8
```

Comment:
You have to add `libgmp-dev` and `libmpfr-dev` here as well as this entry is overriding the above project-wide entry.